protocol/x/listing/keeper/listing.go

```diff
@@ -0,0 +1,35 @@
+package keeper
+
+import (
+	"encoding/json"
+
+	"golang.org/x/xerrors"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/dydxprotocol/v4-chain/protocol/x/listing/types"
+)
+
+// Function to set permissionless listing flag in module store
+func (k Keeper) SetPermissionlessListingEnable(ctx sdk.Context, enable bool) error {
+	store := ctx.KVStore(k.storeKey)
+	b, err := json.Marshal(enable)
+	if err != nil {
+		return err
+	}
+	store.Set([]byte(types.PermissionlessListingEnableKey), b)
+	return nil
+}
+
+// Function to check if permissionless listing is enabled
+func (k Keeper) IsPermissionlessListingEnabled(ctx sdk.Context) (enabled bool, err error) {
```

Comment:
mega nit: `GetPermissionlessListingEnabled`?

Reply:
hmm I kinda like the current better